webapp/src/main/webapp/skin/less/news.less

```diff
@@ -175,13 +175,13 @@
     }
 
     img.pull-right {
-      margin-left: 15px ~'; /** orientation=lt */ ';
-      margin-right: 15px ~'; /** orientation=rt */ ';
+      margin-left: 1.5rem ~'; /** orientation=lt */ ';
```

Comment:
lt and rt to be tested @hassenelkamel-exo

Reply:
yes, well tested